app/uat_actions/uat_actions/generate_tag_layout_template.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
```

Comment:
It's a bit of a shame that the UAT page is an almost exact replica of the existing admin function (/tag_layout_templates/new?tag_group_id=), but I guess it's kind of necessary to fit in with the rest of the UAT / integration test stuff?